ubuntudesign/vanilla-framework
# Add Embedded Media util

## Done

- Added embedded media util
- Deleted redundant helper classes
- Cleaned up naming of utils for consistency
- Added `.u-embedded-media` to replace `.video-container`

## QA

- Sanity check code
- Run code and along side pattern library 
- Check that example Youtube vid displays correctly across breakpoints.

## Details

Fixes #428, #393
